tests/commands/test_debug.py

```diff
@@ -30,6 +34,36 @@ def test__abi_to_build_path():
     assert f"{BUILD_DIRECTORY}/{filename}" == _abi_to_build_path(filename)
 
 
+@pytest.mark.parametrize(
+    "file, address_set",
+    [
+        ([f"{DEBUG_ADDRESS}:{ABI_PATH}"], [int(DEBUG_ADDRESS, 16)]),
+        ([f"{DEBUG_ADDRESS}:{ABI_PATH}:{ALIAS}"], [int(DEBUG_ADDRESS, 16)]),
+    ],
+)
+@patch("nile.utils.debug._abi_to_build_path", return_value=ABI_PATH)
+def test__locate_error_lines_with_abis_with_and_without_alias(
+    mock_path, file, address_set
+):
+    with patch("nile.utils.debug.open") as mock_open:
+        mock_open.return_value.__enter__.return_value = file
+        return_array = _locate_error_lines_with_abis(MOCK_FILE, address_set, mock_path)
+        # The contract alias should be omitted in the output
+        assert return_array == [f"{DEBUG_ADDRESS}:{ABI_PATH}"]
+        assert ALIAS not in return_array[0]
```

Comment:
I'm not sure it's that important that the alias is dropped, but that it works when there's one